document-settings/generatorMode.md

```diff
@@ -23,34 +23,38 @@ Default value:
 
 Div-only
 
-```
+```json
 "settings": {
 	{
 		encodeUrls: true,
-		lang: "en",
+		previewTextHack: false,
 		lineLength: 799,
+		forceHexaColors: true,
```

Comment:
```suggestion
		"forceHexaColors": true,
```

Reply:
There are a lot of similar stuff... won't comment on them anymore :D

Reply:
I think it's because most of the stuff was just copied from the js and I just set everything to a json type without checking. I will fix all of them :)